backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -93,6 +93,16 @@ public DeviceSpecimenType getFirstDeviceSpecimenTypeForDeviceTypeId(UUID deviceI
                     "Device is not configured with a specimen type"));
   }
 
+  public DeviceSpecimenType getDeviceSpecimenType(UUID deviceTypeId, UUID specimenTypeId) {
+    return _deviceSpecimenRepo
+        .findByDeviceTypeInternalIdAndSpecimenTypeInternalIdOrderByCreatedAt(
+            deviceTypeId, specimenTypeId)
+        .orElseThrow(
+            () ->
+                new IllegalGraphqlArgumentException(
+                    "Device is not configured with a specimen type"));
```

Comment:
nit: is this an accurate error message for this?

Reply:
Yeah, its not totally accurate, I'll fix it